interfaces/builtin/mir.go

```diff
@@ -0,0 +1,164 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (c) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more dtails.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var mirPermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the Mir server. Reserved because this
+# gives privileged access to the system.
+# Usage: reserved
+# needed since Mir is the display server, to configure tty devices
+capability sys_tty_config,
+unix (receive, send) type=seqpacket addr=none,
```

Comment:
`unix (receive, send) type=seqpacket addr=none,` should move to mirConnectedSlotAppArmor and be: `unix (receive, send) type=seqpacket addr=none peer=(label=###PLUG_SECURITY_TAGS###),` then you need to adjust mirConnectedSlotAppArmor to substitute \#\#\#PLUG_SECURITY_TAGS\#\#\# like you do  \#\#\#SLOT_SECURITY_TAGS\#\#\# in mirConnectedPlugAppArmor